hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/ipc/TestIdentityProviders.java

```diff
@@ -51,6 +54,9 @@ public int getPriorityLevel() {
     }
   }
 
+  @Rule
+  public ExpectedException exception = ExpectedException.none();
```

Comment:
Looks like CI is not happy with the usage of `ExpectedException` here

Reply:
It seems that it has been deprecated. Replacing it with `assertThrows`, should fix the issue.